src/ert/config/observation_vector.py

```diff
@@ -15,9 +13,9 @@
 
 @dataclass
 class ObsVector:
-    observation_type: EnkfObservationImplementationType
-    observation_key: str
-    data_key: str
+    observation_type: ResponseTypes
+    observation_name: str
+    response_name: str
     observations: Dict[Union[int, datetime], Union[GenObservation, SummaryObservation]]
 
     def __iter__(self) -> Iterable[Union[SummaryObservation, GenObservation]]:
```

Comment:
Not used?

Reply:
Edit: Turns out they were both used afterall